lib/govuk_design_system_formbuilder/elements/error_summary.rb

```diff
@@ -65,6 +65,8 @@ def error_summary_title_id
       end
 
       def object_has_errors?
+        return unless @builder.object.respond_to?(:errors)
```

Comment:
I think the builder _should_ error if the dev tries to render an error summary in a form with an object that doesn't support errors

Reply:
> Overall this looks good. Before this was raised I'd already created #90 which does more or less the same thing. If you're happy for me to I'll cherry pick the tests from this PR into that one and tweak accordingly. Thanks @jsugarman

Yep, what ever you think is best.

Reply:
> I think the builder _should_ error if the dev tries to render an error summary in a form with an object that doesn't support errors

👍 Makes sense, better to catch these things early.